src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaCollection.js

```diff
@@ -47,7 +47,6 @@ class MediaCollection extends React.Component<Props> {
     @action handleUpload = (media: Array<Object>) => {
         const {mediaListStore} = this.props;
 
-        mediaListStore.reset();
```

Comment:
calling `reset` here was necessary to prevent the media list from displaying the last loaded page only. with the changes in this pr, this is not necessary anymore